Sites/GD2/Project 1/Styles.css

```diff
@@ -0,0 +1,105 @@
+img {
+    max-width: 100%
+}
+
+h1 {
+    color: #260648;
+    font-weight: bold;
+    font-size: 40px
+}
+
+bodycolor {
+    color: #F6F6EC
+}
+
+h2 {
+    color: #6B4B8C;
+    font-weight: bold;
+    font-size: 20px;
+    font-alignment: left
+}
+
+div {
+    margin: 12px
+}
+
+width=39em p {
+    color: #1C0534;
+}
+
+a {
+    color: #EE3595;
+}
+
+a:hover {
+    text-decoration: underline;
+
+    /* Will underline all links when the user puts their mouse over them */
+}
+
+p1 {
+    font-weight: regular;
+    border: 1px solid black;
+}
+
+p2 {
+    font-weight: regular;
+    border: 1px solid black;
+    text-align: center;
+}
+
+h1 {
+    font-family: Futura;
+    font-weight: bold;
+}
+
+h2 {
+    font-family: Futura;
+}
+
+p {
+    font-family: Garamond;
+}
+
+figcaption {
+    font-family: Garamond;
+}
+
+h {
```

Comment:
Everything from here to line 104 can be deleted, as it's not valid CSS. We'll talk on Tuesday about what you're looking to achieve. 👍🏻